### ISSUE TITLE
cannot open shared object file: libgstreamer-0.10.so.0

### ISSUE THREAD

**Issue by hellocatfood** [nuigroup/ccv2#26]
I'm attempting to launch this software on Ubuntu 16.04 and I'm encountering an error when I run ./clickToLaunchApp.sh

`./Community_Core_Vision: error while loading shared libraries: libgstreamer-0.10.so.0: cannot open shared object file: No such file or directory`

I have checked and I already have installed libgstreamer0.10-dev and libgstreamer0.10-0-dev. Is there a fix for this?

**Comment by int-ua** [comment]
Do you have `libgstreamer0.10-0` installed?

**Comment by hellocatfood** [comment]
Yes, I do (sorry, I posted the same library name twice in my earlier comment)

**Comment by int-ua** [comment]
Sorry, still not clear, do you have the version without `-dev` in the name?

**Comment by hellocatfood** [comment]
I have the following installed `libgstreamer1.0-dev libgstreamer-plugins-bad1.0-0 libgstreamer-plugins-bad1.0-0 libgstreamer-plugins-base0.10-0 ibgstreamer-plugins-base1.0-dev libgstreamer0.10-dev libgstreamer0.10-0`

**Comment by int-ua** [comment]
Maybe it tries to open a 32bit version?
Can you execute `file ./Community_Core_Vision`

**Comment by hellocatfood** [comment]
Output of `file ./Community_Core_Vision`

`./Community_Core_Vision: ELF 32-bit LSB executable, Intel 80386, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux.so.2, for GNU/Linux 2.6.15, stripped`

**Comment by int-ua** [comment]
It's 32-bit and if your system is 64-bit

```
sudo apt install libgstreamer0.10-0:i386
```

should fix the problem

**Comment by tejasrinivasraj** [comment]
sudo dnf install gstreamer gstreamer-plugins-base-devel

This seems working!
